### PR TITLE
enh(UI): Improve generic properties

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This improves the generic properties.

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by Noha-ElAbrouki — APPROVED** [review]

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16172,3 +16172,12 @@ msgstr ""
 
 # msgid "The limit of 2 units has been reached"
```
Comment (smau-centreon) [inline centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:16173]:
```suggestion
# msgid "You have reached the limit of 2 units"
```

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -15837,3 +15837,12 @@ msgstr ""
...
 # msgid "The limit of 2 units has been reached"
 # msgstr ""
+
+# msgid "Show description"
+# msgstr ""
+
+# msgid "Open links in new tab"
+# msgstr ""
+
+# msgid "By turning off this option, all links included in the widget will be opened in the same tab when clicked."
```
Comment (smau-centreon) [inline centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:15847]:
```suggestion
# msgid "By turning off this option, all the links included in the widget will be opened in the same tab."
```

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -15837,3 +15837,12 @@ msgstr ""
 
 # msgid "The limit of 2 units has been reached"
```
Comment (smau-centreon) [inline centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:15838]:
```suggestion
# msgid "You have reached the limit of 2 units"
```

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -16172,3 +16172,12 @@ msgstr ""
...
 # msgstr ""
+
+# msgid "Show description"
+# msgstr ""
+
+# msgid "Open links in new tab"
+# msgstr ""
+
+# msgid "By turning off this option, all links included in the widget will be opened in the same tab when clicked."
```
Comment (smau-centreon) [inline centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:16182]:
```suggestion
# msgid "By turning off this option, all the links included in the widget will be opened in the same tab."
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -18005,3 +18005,12 @@ msgstr "Une erreur est survenue lors de la récupération des données de métri
...
 msgid "Invalid metric format"
 msgstr "Format de métrique invalide"
+
+msgid "Show description"
+msgstr "Afficher la description"
+
+msgid "Open links in new tab"
+msgstr "Ouvrir les liens dans un nouvel onglet"
+
+msgid "By turning off this option, all links included in the widget will be opened in the same tab when clicked."
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18015]:
```suggestion
msgid "By turning off this option, all the links included in the widget will be opened in the same tab."
```

File: centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -18005,3 +18005,12 @@ msgstr "Une erreur est survenue lors de la récupération des données de métri
...
+msgstr "En désactivant cette option, tous les liens inclus dans le widget seront ouverts dans le même onglet lorsqu'ils sont cliqués."
```
Comment (smau-centreon) [inline centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:18016]:
```suggestion
msgstr "En désactivant cette option, tous les liens inclus dans le widget seront ouverts dans le même onglet."
```

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -16636,3 +16636,12 @@ msgstr ""
 
 # msgid "The limit of 2 units has been reached"
```
Comment (smau-centreon) [inline centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:16637]:
```suggestion
# msgid "You have reached the limit of 2 units"
```

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -16636,3 +16636,12 @@ msgstr ""
...
 # msgstr ""
+
+# msgid "Show description"
+# msgstr ""
+
+# msgid "Open links in new tab"
+# msgstr ""
+
+# msgid "By turning off this option, all links included in the widget will be opened in the same tab when clicked."
```
Comment (smau-centreon) [inline centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:16646]:
```suggestion
# msgid "By turning off this option, all the links included in the widget will be opened in the same tab."
```

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 9]
```diff
@@ -16625,3 +16625,12 @@ msgstr "Você não salvou as alterações, deseja continuar?"
 
 # msgid "The limit of 2 units has been reached"
```
Comment (smau-centreon) [inline centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:16626]:
```suggestion
# msgid "You have reached the limit of 2 units"
```

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 10]
```diff
@@ -16625,3 +16625,12 @@ msgstr "Você não salvou as alterações, deseja continuar?"
...
 # msgstr ""
+
+# msgid "Show description"
+# msgstr ""
+
+# msgid "Open links in new tab"
+# msgstr ""
+
+# msgid "By turning off this option, all links included in the widget will be opened in the same tab when clicked."
```
Comment (smau-centreon) [inline centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:16635]:
```suggestion
# msgid "By turning off this option, all the links included in the widget will be opened in the same tab."
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Comment by smau-centreon** [comment]
@Thebarda Where is the FR translation for "You have reached the limit of 2 units"?

**Comment by Thebarda** [comment]
> @Thebarda Where is the FR translation for "You have reached the limit of 2 units"?

Will add it

**Comment by smau-centreon** [comment]
> > @Thebarda Where is the FR translation for "You have reached the limit of 2 units"?
> 
> Will add it

Will be "Vous avez atteint la limite de 2 unités"

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po (hunk 1 above)
Comment (Thebarda) [reply centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:16173]: Done in this upcoming PR: https://github.com/centreon/centreon/pull/2029/files#diff-dc289dcc37d296608a03cf9223221a68a496ee20a5ab3e83d37bb2a202040112

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po (hunk 3 above)
Comment (Thebarda) [reply centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:15838]: Same comment

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po (hunk 7 above)
Comment (Thebarda) [reply centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:16637]: Same comment

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po (hunk 9 above)
Comment (Thebarda) [reply centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:16626]: Same comment

**Review by smau-centreon — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by jeremyjaouen — APPROVED** [review]

**Review by abuathier — APPROVED** [review]

**Review by Noha-ElAbrouki — APPROVED** [review]

**Pull request merged**